### PR TITLE
removed getProps from FloatingButton tests

### PR DESCRIPTION
**Pull request by nitzanyiz**
## Description
Removed getProps from FloatingButton tests.

## Changelog
None

## Additional info
None

### CODE REVIEW

File: src/components/floatingButton/__tests__/index.spec.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const TestCase = (props) => {
 };
 export const FloatingButtonDriver = (props: ComponentProps) => {
   const driver = useComponentDriver(props);
-  const getStyle = () => driver.getProps().style as ViewStyle;
+  const getStyle = () => driver.getElement().props.style as ViewStyle;
```
Comment (Inbal-Tish) [inline src/components/floatingButton/__tests__/index.spec.tsx:22]: Isn't props optional? Don't you need to do `driver.getElement().props**?**.style`?

Comment (nitzanyiz) [reply src/components/floatingButton/__tests__/index.spec.tsx:22]: I think they're not. Its always a component that is captured so it has testId so props exists.

Comment (nitzanyiz) [reply src/components/floatingButton/__tests__/index.spec.tsx:22]: Plus according to the type `ReactTestInstance` props is not optional so they probably promise us there will be props.

Comment (Inbal-Tish) [reply src/components/floatingButton/__tests__/index.spec.tsx:22]: Ok...

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**